sprint8/src/pages/free-board/create-post.js

```diff
@@ -0,0 +1,65 @@
+import { useState, useEffect } from 'react';
+import { useRouter } from 'next/router';
+import style from '@/src/styles/create-post.module.css';
+import Header from '@/src/components/Header.js';
+import Button from '@/src/components/Button.js';
+import CreateEditPost from '@/src/components/CreateEditPost.js';
+import axios from '@/src/lib/axios.js';
+
+export default function CreatePost() {
+  const router = useRouter();
+  const [isButtonActive, setIsButtonActive] = useState(false);
+  const [title, setTitle] = useState('');
+  const [content, setContent] = useState('');
+
+  async function postArticle() {
+    const data = { title: title, content: content };
```

Comment:
@annylee0723 자바스크립트에서 object literal 방식으로 객체를 만들때 key와 value가 동일하다면 key만 입력해도돼요
https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Object_initializer

```suggestion
    const data = { title, content };
```

Reply:
수정 완료했습니다 !
commit: [f27c2d6](https://github.com/codeit-sprint-fullstack/2-Sprint-Mission-FE/pull/148/commits/f27c2d6064ed42b50e6bf559d9181b49bd5ac044)

Reply:
@annylee0723 넵 좋습니다!